### ISSUE TITLE
Database creation script

### ISSUE THREAD

**Issue by hanskhe** [magination/blog#50]
We need a script that creates a test DB, that is run as part of the `npm install` process. It should be kept up to date with any changes to the DB model, so we don't need to manually fix our dev environment DBs.

**Comment by tiborv** [comment]
Yes, this ties in with the proper setting of NODE_ENV to get an distinction between dev/prod envs.
For the schema changes/updates, our easiest option is to just dump the test db and repopulate it. We could also consider creating migration scripts with something like mongoose-migrate, but I don't think this is worth doing for a small project like this.

**Comment by hanskhe** [comment]
I agree that some of this might be overkill for a small project, but I think that some handling of changes to the DB in the dev env. should be done. I have problems testing changes because my local DB is out of date. It will also be great learning for later projects. 

I think dumping and repopulating will work great for this project.

**Comment by tiborv** [comment]
Yes, I agree. Dump+repopulate. 

Should this maybe be an optional gulp task? There could be some "funny" stuff happening with nodemon always restarting (causing the dump to happen each time).
